### PR TITLE
feat(pkger): adds gauge chart support

### PR DESCRIPTION
**Pull request by dearyhud**
Screenshot after applying gauge chart via pkger CLI

Screen Shot 2019-11-05 at 4 35 59 PM

Closes #

Describe your proposed changes here.

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: pkger/models.go [hunk 1]
```diff
@@ -547,6 +562,8 @@ func (c chart) validProperties() []failure {
 		fails = append(fails, c.Colors.hasTypes(colorTypeScale)...)
 		fails = append(fails, validGeometry(c.Geom)...)
 		fails = append(fails, c.Axes.hasAxes("x", "y")...)
+	case ChartKindGauge:
+		fails = append(fails, c.Colors.hasTypes(colorTypeMin, colorTypeThreshold, colorTypeMax)...)
```
Comment (jsteenb2) [inline pkger/models.go]: are all these colors required for the chart to showup? or are some optional and we have graceful fallbacks? if they are required, lets keep it this as is 👌

File: pkger/testdata/dashboard_gauge.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        - kind:   gauge
+          name:   gauge
+          note: gauge note
+          noteOnEmpty: true
+          xPos:  1
+          yPos:  2
+          width:  6
+          height: 3
+          shade: true
+          geom: line
```
Comment (jsteenb2) [inline pkger/testdata/dashboard_gauge.yml]: is geom required for this?

File: pkger/testdata/dashboard_gauge.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          legend:
```
Comment (jsteenb2) [inline pkger/testdata/dashboard_gauge.yml]: axe legend not applicable. Lets keep these files to only the vital data that is useful to a guage cahrt. axe the rest, is a bit confusing to see here and not above

File: pkger/models.go (hunk 1 above)
Comment (dearyhud) [reply pkger/models.go]: they're all required

File: pkger/testdata/dashboard_gauge.yml (hunk 2 above)
Comment (dearyhud) [reply pkger/testdata/dashboard_gauge.yml]: no, copy-pasta

File: pkger/testdata/dashboard_gauge.yml (hunk 3 above)
Comment (dearyhud) [reply pkger/testdata/dashboard_gauge.yml]: same here - copy-pasta

Comment (jsteenb2) [reply pkger/testdata/dashboard_gauge.yml]: right on, go on and axe the erroneous fields and we'll be ready to rock n roll here 👍

**Pull request merged**